pegen/parse_string.c

```diff
@@ -1195,7 +1197,6 @@ make_str_node_and_del(Parser *p, PyObject **str, Token* first_token, Token *last
         return NULL;
     }
     //TODO: Check this logic with the kind
-    //
     const char* the_str = PyBytes_AsString(first_token->bytes);
```

Comment:
Can we remove this TODO now?

Reply:
Yep, I will do another PR after this one to make the "u" an interned string so we don't create one every time.